plugins/TelescopeControl/src/ASCOM/TelescopeClientASCOM.cpp

```diff
@@ -28,26 +28,7 @@
 #include "StelUtils.hpp"
 #include "StelTranslator.hpp"
 
-bool useJNow(ASCOMDevice::ASCOMEquatorialCoordinateType coordinateType, bool mAscomUseDeviceEqCoordType, Equinox mEquinox)
```

Comment:
the system file format is the act of the mirrors

Reply:
I have move this into a static method of the class that uses it (encapsulation). Can you elaborate what problem you see?

Reply:
actually in the mathematical form which counted the line graph form, it reduces on the line methods on that mathematical form but on codding which you use the encapsulation the format of files which uses on graph methods has the problem on whole of the mathematical which changes the lines of flows as on physical and mathematical on both, the system file of boolian numbers intelligences as the real local form of lines and on the other hand the shape method has the static form as you know, but when it make on statistical form the base of the statics has the both problem as in this side StelUtils.hpp and StelTranslator.hpp too on the mirror side, i mean At the spase side of Coding we use the path directories but on the AI form we use the both of file formats as the space of directory and the definition of the both.

Reply:
Seems you are wasting my time.